### ISSUE TITLE
Last commit broke the project

### ISSUE THREAD

**Issue by dsoprea** [go-errors/errors#27]
Your code isn't currently compiling. The tests are reflecting this. Looks like the last change broke it.

https://travis-ci.org/github/go-errors/errors/jobs/692460222

![image](https://user-images.githubusercontent.com/715626/83373953-19b6fa80-a398-11ea-89a7-ada2f4b18c93.png)

https://github.com/go-errors/errors/commit/baa9282cdbe8881041921eca5aa81e4805054f8e#diff-faeee970b41138d8bdc51e7aa3601141R147

You probably meant `Is()`, not `errors.Is()`.

Please fix or revert.

**Comment by dsoprea** [comment]
Incidentally, it looks like you aren't running the unit-tests on pull-requests. Otherwise, that could've allowed you to identify this before merging.

https://github.com/go-errors/errors/pull/26

**Comment by ConradIrwin** [comment]
Thanks for reporting @dsoprea, it looks like the tests are running `go1.10` which has been unsupported for a while now.

We introduced a fix that only works on `go1.13` (the oldest maintained version of go) and above; what version of go are you running?

@titouanfreville I think it should be easy to make your change backward compatible by using go build tags to conditionally change the implementation of `Is` depending on the go version, or by copy-pasting the code from go1.13's errors package into here. Is that something you'll have time to look at?

**Comment by dsoprea** [comment]
I hadn't realized that several of my projects weren't recognized as
modulized in Travis. So, I was running against your master tip. I forced
module support, disabled 1.10 and 1.11, and got unblocked.

I would encourage you to revert immediately as your project is currently
unusable for first-time implementers. Put the current version in a branch
for them to fix. Once fixed, squash and rebase back to master.

**Comment by titouanfreville** [comment]
Implemented solution with tags. Test are no ok. 
Would be nice to have a lock on PR when travis fails.

**Comment by dsoprea** [comment]
It's a project setting under "Webhooks".

![Screenshot_20200602-105711_Chrome](https://user-images.githubusercontent.com/715626/83535636-32bacb00-a4c0-11ea-9f10-631068dd38ad.jpg)
